### PR TITLE
Issue1 db brand

### PR DESCRIPTION
**Pull request by RealDogDad**
###Demo Ready Database Branch
## It will do the following without UI
---
1. Create a DB instance
2. Create an AccountEntity
3. Update the AccountBalance
4. Delete the AccountEntity
---
This will be demonstrated by Toast notifications.

### CODE REVIEW

**Review by Dylan-Riley — APPROVED** [review]
Thanks for getting this sorted out! Excited to finally see the DB and UI come together

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.0.1"
+        classpath 'com.android.tools.build:gradle:4.1.0'
```
Comment (jleeboyd) [inline build.gradle:8]: Is there a specific reason you switched to 4.1.0?

Comment (jleeboyd) [reply build.gradle:8]: Does the app run on your machine with this version number? My machine throws errors when using 4.1.0.

**Review by jleeboyd — APPROVED** [review]
The demo works great on my en. I see you have two other toasts, however I only ever see this one. Is this by design?

Screen Shot 2020-10-26 at 9 33 02 PM

File: build.gradle (hunk 1 above)
Comment (Dylan-Riley) [reply build.gradle:8]: That is the most recent version of gradle IIRC. Android Studio's been harassing me to upgrade but I've been too paranoid to do so

Comment (jleeboyd) [reply build.gradle:8]: Hm we should determine which version is best for our project. I was having problems with 4.1.0, but if @RealDogDad has found a way to make it work we should go with that.

Comment (RealDogDad) [reply build.gradle:8]: Oh interesting catch, I will look and see if the update breaks it.

**Pull request merged**